packages/libs/react-ui/src/components/Divider/Divider.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import { Divider } from './Divider';
+
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+
+const meta: Meta = {
+  title: 'Layout/Divider',
+  component: Divider,
+};
+
+type Story = StoryObj;
+
+export const Static: Story = {
+  name: 'Divider',
+
+  render: () => {
+    return <Divider />;
```

Comment:
Can you add some context for the divider? Like content to show how it would be used